lib/services/db.js

```diff
@@ -215,6 +216,91 @@ function formatBatchOperations(ops) {
   }).join('\n');
 }
 
+/**
+ *
+ * @param {object} hookOps
+ * @param {Array} hookOps.put
+ * @param {Array} hookOps.del
+ * @param {object} op
+ * @returns {object}
+ */
+function addToHookOps(hookOps, op) {
+  if (hookOps[op.type]) {
+    hookOps[op.type].push(_.pick(op, 'type', 'key', 'value'));
```

Comment:
Good catch, _.pick was left over from when I was removing the `type` property, but now _.pick is no longer needed.

concat creates a new Array, which is not necessary / is not as performant.